### PR TITLE
Start the controller chapter of the CMS tutorial.

### PR DESCRIPTION
**Pull request by markstory**
Another small section for the new tutorial

### CODE REVIEW

**Comment by markstory** [comment]
@antograssiot No it wasn't on purpose a bunch of this content was pulled from the blog tutorial, or I forgot..

File: en/tutorials-and-examples/cms/articles-controller.rst [hunk 1]
```diff
@@ -1,9 +1,132 @@
...
+    class ArticlesController extends AppController
+    {
+    }
+
+Now, let's add an action to our controller. Actions are controller methods that
+have routes connected to them. For example, when a user requests
+**www.example.com/articles/index** (which is also the same as
+**www.example.com/articles**), CakePHP will call the ``index`` method of your
+``ArticlesController``. This method should query the model layer, and prepare
+a response (via the View and Template). The code for that action would look like
```
Comment (ravage84) [inline en/tutorials-and-examples/cms/articles-controller.rst]: "a response by rendering the View and Template", may be?

File: en/tutorials-and-examples/cms/articles-controller.rst [hunk 2]
```diff
@@ -1,9 +1,132 @@
...
+
+    You may be tempted to name your controllers and actions in a way that allows
+    you to obtain specific URLs. Resist that temptation. Instead, follow the
+    :doc:`/intro/conventions` creating readable, meaningful action names. You
+    can then use doc:`/development/routing` to connect the URLs you want to the
+    actions you've created.
+
+Our controller action is very simple. It fetches all the articles from the
+database, using the model that is automatically loaded via naming conventions.
+It then uses ``set()`` to pass the articles into the view (which we'll create
```
Comment (ravage84) [inline en/tutorials-and-examples/cms/articles-controller.rst]: Here, "view" is in lowercase, [above](https://github.com/cakephp/docs/pull/4870/files#diff-51a543267d06defd6019191b3b648034R23) it's in uppercase. 😕

File: en/tutorials-and-examples/cms/articles-controller.rst [hunk 3]
```diff
@@ -1,9 +1,132 @@
...
+    }
+
+By defining function ``index()`` in our ``ArticlesController``, users can now
+access the logic there by requesting **www.example.com/articles/index**. Similarly,
+if we were to define a function called ``foobar()``, users would be able to
+access that at **www.example.com/articles/foobar**.
+
+.. warning::
+
+    You may be tempted to name your controllers and actions in a way that allows
```
Comment (ravage84) [inline en/tutorials-and-examples/cms/articles-controller.rst]: While I think it's good to have this warning, I feel it pulls away the concentration of a reader. Could we move this below the next  paragraph?

File: en/tutorials-and-examples/cms/articles-controller.rst [hunk 4]
```diff
@@ -1,9 +1,132 @@
...
+You might have noticed the use of an object called ``$this->Html``.  This is an
+instance of the CakePHP :doc:`HtmlHelper </views/helpers/html>`.  CakePHP comes
+with a set of view helpers that make tasks like creating links, forms, and
+pagination buttons easy. You can learn more about :doc:`/views/helpers` in their
+chapter, but what's important to note here is that the ``link()`` method will
+generate an HTML link with the given link text (the first parameter) and URL
+(the second parameter).
+
+When specifying URLs in CakePHP, it is recommended that you use arrays or
+:ref:`named routes <named-routes>`. Using these forms for URLs allows you to
```
Comment (ravage84) [inline en/tutorials-and-examples/cms/articles-controller.rst]: What could be an alternatie to "forms"? In the paragraph above we already talk about forms. Readers who aren't proficient in English could be confused.

File: en/tutorials-and-examples/cms/articles-controller.rst (hunk 1 above)
Comment (markstory) [inline en/tutorials-and-examples/cms/articles-controller.rst]: Good idea.

File: en/tutorials-and-examples/cms/articles-controller.rst (hunk 3 above)
Comment (markstory) [inline en/tutorials-and-examples/cms/articles-controller.rst]: I'll remove the call out and make it part of the normal text.

File: en/tutorials-and-examples/cms/articles-controller.rst (hunk 4 above)
Comment (markstory) [inline en/tutorials-and-examples/cms/articles-controller.rst]: I'll use 'syntax'

**Pull request merged**